.github/workflows/ci.yml

```diff
@@ -199,6 +199,50 @@ jobs:
         command: test
         args: --no-run --test codegen beat_codegen
 
+  coverage:
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis
+        ports:
+          - 6379:6379
+      rabbitmq:
+        image: rabbitmq
+        ports:
+          - 5672:5672
+        env:
+          RABBITMQ_DEFAULT_VHOST: my_vhost
+    steps:
+    - name: Install rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: nightly
+        override: true
+
+    - uses: actions/checkout@v2
+
+    - name: Ready cache
+      run: sudo chown -R $(whoami):$(id -ng) ~/.cargo/
+
+    - name: Cache cargo
+      uses: actions/cache@v1
+      id: cache
+      with:
+        path: ~/.cargo
+        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
```

Comment:
Is there a reason for doing it this way instead of how we do set up the cache in the other jobs?

https://github.com/rusty-celery/rusty-celery/blob/f73ae039a95a268eb1df475d7597c252ea5456b0/.github/workflows/ci.yml#L68-L85

Reply:
not at all, I am going to change that